core/src/test/java/dk/alexandra/fresco/lib/real/LinearAlgebraTest.java

```diff
@@ -0,0 +1,358 @@
+package dk.alexandra.fresco.lib.real;
+
+import dk.alexandra.fresco.framework.Application;
+import dk.alexandra.fresco.framework.DRes;
+import dk.alexandra.fresco.framework.builder.numeric.ProtocolBuilderNumeric;
+import dk.alexandra.fresco.lib.collections.Matrix;
+import dk.alexandra.fresco.lib.collections.MatrixUtils;
+import org.junit.Test;
+
```

Comment:
Please reorder imports to conform to style. Single block ordered alphabetically (static imports go in a seperate block though) (https://google.github.io/styleguide/javaguide.html#s3.3-import-statements)

Reply:
Yes, of course. I reordered them in new commit.